Core.ApplicationServices/Contract/IItContractService.cs

```diff
@@ -25,5 +25,6 @@ public interface IItContractService
         Maybe<OperationError> ValidateNewName(int contractId, string name);
         IQueryable<ItContract> Query(params IDomainQuery<ItContract>[] conditions);
         Result<ContractOptions, OperationError> GetAssignableContractOptions(int organizationId);
+        Result<IEnumerable<(int year, int quarter)>,OperationError> GetAppliedProcurementPlans(int organizationId);
```

Comment:
@a-naskret Notica that before the changes, this one allowed any one to query any contract info.. we must always apply access control in the application layer